landscape/client/monitor/snapmonitor.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from landscape.client.monitor.plugin import DataWatcher
+from landscape.client.snap.http import SnapdHttpException
+from landscape.client.snap.http import SnapHttp
+
+
+class SnapMonitor(DataWatcher):
+
+    run_interval = 1800  # 30 minutes
```

Comment:
Seems like this value is a always overwritten?

Reply:
Right. I think I forgot to delete it when I added the config var. Will remove.